frontend/src/components/Footer.vue

```diff
@@ -7,13 +7,14 @@
         alt="logo"
         height="53.02px"
       />
-
-      <img
-        class="footer__logos-hyf"
-        src="../assets/hyf_logo.png"
-        alt="HYF"
-        height="53.02px"
-      />
+      <div>
+        <img
+          class="footer__logos-hyf"
```

Comment:
this image becomes too small in the mobile version, maybe we could just delete it ?